core/src/main/scala/org/apache/spark/ui/SparkUI.scala

```diff
@@ -50,6 +50,7 @@ private[spark] class SparkUI private (
     val operationGraphListener: RDDOperationGraphListener,
     var appName: String,
     val basePath: String,
+    val lastUpdateTime: Long = -1L,
```

Comment:
I would like to user `Option[Long] = None` as default value to reflect there's no update time.

Reply:
Update @jerryshao Thanks for your time.